core/queue/views.py

```diff
@@ -49,6 +52,9 @@ def retrieve(self, request, program_id=None):
                 visit_event_data = FrontDeskEventForQueueSerializer(
                     front_desk_events_queryset, many=True
                 ).data
+                #visit_event_data = FrontDeskEventForQueueSerializer(
+                #    front_desk_events_queryset, many=True, context={'request': request}
+                #).data
```

Comment:
Rather than cut lines 55-57, I cut 52-54. As above, there should be a context, although it didn't throw an error. In my opinion it should and I would like it to be consistent with the case above